tools/pipelines/test-real-service.yml

```diff
@@ -59,7 +59,7 @@ stages:
         testPackage: ${{ variables.testPackage }}
         testWorkspace: ${{ variables.testWorkspace }}
         artifactBuildId: $(resources.pipeline.client.runID)
-        testCommand: test:realsvc:tinylicious:report:full
+        testCommand: test:realsvc:tinylicious:report
```

Comment:
What about ODSP and FRS? (and r11s...?)

Reply:
We don't run the full compat matrix against any of those, just certain variants (lines 88-96, 118-126, 148-159 below).